motion_dualtask/dataset_description.json

```diff
@@ -0,0 +1,19 @@
+{
+	"Acknowledgements":"n\/a",
```

Comment:
why are the slashes escaped?

Reply:
It has to do with JSONLab, which I used for creating this dataset. Apparently the official JSON syntax allows it, too, but I had suggested an update to skip escape. Please see [this thread ](https://github.com/fangq/jsonlab/issues/82).

Reply:
The thread tells me that you found a solution, are you still planning to propagate the solution here?